javascript/house-style.md

```diff
@@ -343,51 +342,33 @@ We use automated tools (like Snyk) to monitor the status of dependencies in our
 
 We use white space liberally to help keep code readable. You're encouraged to use newlines to break up long functions into logical chunks.
 
-You should also remove trailing white space from lines of code. Your editor should be able to either remove this or highlight it when present, for example, [atom-whitespace](https://github.com/atom/whitespace).
+You should also remove trailing white space from lines of code. Your editor should be able to either remove this or highlight it when present, for example, [atom-whitespace](https://github.com/atom/whitespace), [vscode-trailing-spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces).
 
-### Semi-colons
+### Variables
 
-We require the use of semi-colons. The only statements that don't need to end with semi-colons are function declarations and blocks.
+#### Use of const and let
+We define variables with multiple `const` and `let` statements.
 
 We do this:
 
 ```js
-var foo = 1;
-
-if (foo) {
-    foo += 1;
-}
-
-function bar() {
-    // ...
-}
-
-var baz = function() {
-    // ...
-};
+const foo = 1;
+const bar = 2;
 ```
 
 We _don't_ do this:
 
 ```js
-var foo = 1
-
-if (foo) {
-    foo += 1
-};
-
-function bar() {
-    // ...
-};
-
-var baz = function() {
-    // ...
-}
+const foo = 1,
+      bar = 2;
 ```
 
-### Variables
+We use `let` only when a variable _explicitly_ [needs to be mutable](https://ada.is/blog/2015/07/13/immutable/):
```

Comment:
Is there a missing example here? The colon on the end suggests so 😉

Reply:
Ah yes, I found the example given a bit meaningless.

```js	
let foo = 'bar';	
```

Reply:
Maybe this is an ok example?

We do this:

```js
let foo = 1;

if (foo) {	
    foo += 1;	
}
```

```js
const foo = [];

if (bar) {
    foo.push(1);
}
```

We _don't_ do this:

```js
let foo = [];

if (bar) {
    foo.push(1);
}
```

Reply:
maybe using a loop would be a good way to use a more meaningful example? e.g.

```
for(let i = 0; i < 10; i++)  { }
```

I'm not wedded to this suggestion, though - I'm only making it because it would make sense to have an example if `const` and `let` had their own headings! But they don't _need_ separate headings, especially if only one has a meaningful code example 🤷‍♀️

Reply:
oh you beat me to the comment 😄 

Yeah, i like those examples!

Reply:
I actually changed the heading to "Defining variables".
What do you think? https://github.com/springernature/frontend-playbook/pull/268/commits/356330ab79cfc15e0d0e3ba5d282c4111643768f

Reply:
yeah, nicely done 👍